mlflow/projects/__init__.py

```diff
@@ -724,7 +724,8 @@ def _parse_kubernetes_config(backend_config):
         raise ExecutionException("Could not find 'kube-job-template-path': {}".format(
             kube_job_template))
     if 'kube-context' not in backend_config.keys():
-        raise ExecutionException("Could not find kube-context in backend_config.")
+        _logger.debug("Could not find kube-context in backend_config."
```

Comment:
@dbczumar @ReDucTor, since the configuration is valid even without 'kube-context', I made it a debug log instead